src/home.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Navigation } from 'react-native-navigation';
+import { getUserToken } from './components/cache';
+
+const Home = () => {
```

Comment:
```suggestion
export const HomePage = () => {
```
e ai, se for usar assim, pode tirar o `export default`